pinot-segment-local/src/main/java/org/apache/pinot/segment/local/aggregator/DistinctCountCPCSketchValueAggregator.java

```diff
@@ -34,11 +35,11 @@ public class DistinctCountCPCSketchValueAggregator implements ValueAggregator<Ob
   private final int _lgK;
 
   public DistinctCountCPCSketchValueAggregator(List<ExpressionContext> arguments) {
```

Comment:
Do we need to change `DistinctCountCPCSketchAggregationFunction` or sketches with different `lgK` are mergable?

Reply:
@yashmayya The theta related query aggregation function uses nominal entries as a parameter and this same approach was used for the CPC sketch for consistency.  A use case for the lgK parameter on the StarTree for some sketches (eg. Theta, CPC, Tuple) is to lower the accuracy and storage footprint for older data.